lib/matplotlib/backends/backend_gtk.py

```diff
@@ -1003,13 +999,13 @@ def on_combobox_lineprops_changed(self, item):
         if marker is None: marker = 'None'
         self.cbox_markers.set_active(self.markerd[marker])
 
-        r,g,b = colorConverter.to_rgb(line.get_color())
-        color = gtk.gdk.Color(*[int(val*65535) for val in (r,g,b)])
+        rgba = mcolors.to_rgba(line.get_color())
```

Comment:
@fariza is the GTK expert.

There are some applications which do not know / use / care about alpha, forcing all of them to explicitly drop the last value is falling a bit too far to the 'purity' side of API design.